web/controllers/github_events_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule CodeCorps.GitHubEventsController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.GitHub.Webhook.Processor
+
+  @github_event_header_name Application.get_env(:code_corps, :github_event_header_name)
+
+  def create(conn, event_payload) do
+    event_type = conn |> extract_event_type()
+    case event_type |> check_support() do
+      :supported ->
+        Processor.process_async(event_type, event_payload)
+        conn |> send_resp(200, "")
+      :unsupported ->
+        conn |> send_resp(202, "")
+    end
+  end
+
+  defp check_support(event) when event in ~w(installation
+    installation_repositories issue_comment issues), do: :supported
+  defp check_support(event) when event in ~w(commit_comment create delete
+    deployment deployment_status fork gollum label marketplace_purchase
+    member membership milestone organization org_block page_build
+    project_card project_column project public pull_request_review_comment
+    pull_request_review pull_request push repository release status team
+    team_add watch), do: :unsupported
+  defp check_support(_event), do: :unsupported
```

Comment:
This looks gross but also okay with proceeding unless there's a quick change to make it better.